index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Ahmad Shah Baba</h1>
+      <div id="img-div">
+        <img id="image" src="C:\Users\Power link\Downloads\images.jpg" alt="Ahmad Shah Baba" class="centered-image" align="center">
```

Comment:
- [SUGESSION] - As I can see your image is not showing up in the browser and the problem is that you added the image from your computer and provided the folder link from your computer which is wrong.
  - To solve this, add the image to your project main directory and then use the image folder link in href.